javascripts/vendor/tiny-totp.js

```diff
@@ -0,0 +1,158 @@
+/*
```

Comment:
Should this be added to `package.json` and add it to `lib/tasks/javascript.rake` too?